util/src/test/java/io/kubernetes/client/CopyTest.java

```diff
@@ -127,4 +129,53 @@ public void run() {
             .withQueryParam("command", equalTo("-c"))
             .withQueryParam("command", equalTo("base64 -d | tar -xmf - -C /")));
   }
+
+  @Test
+  public void testCopyDirectoryFromPod() throws IOException, ApiException, InterruptedException {
+
+    File testFile = File.createTempFile("testfile", null);
+    testFile.deleteOnExit();
+
+    Copy copy = new Copy(client);
+
+    stubFor(
+        get(urlPathEqualTo("/api/v1/namespaces/" + namespace + "/pods/" + podName + "/exec"))
+            .willReturn(
+                aResponse()
+                    .withStatus(404)
+                    .withHeader("Content-Type", "application/json")
+                    .withBody("{}")));
```

Comment:
ahh, so sry for oversight.. the exec call works basing on websocket, there's no way to mock that out via wiremock framework.. i will verify this pull locally. we should remove the test

Reply:
Let me remove the test.

Reply:
Test case is removed.